docs/faq.md

```diff
@@ -16,3 +16,15 @@ Exact URLs used are listed in the table below:
 
 > Note: The Generic component loads the right widget as per its selected view type
 
+### I am seeing some funny characters in the HTML rendering of my widget
+
+The Stylitics widgets require the HTML page to have UTF-8 encoding enabled. Simply add the following to the `<head />` of your HTML.
```

Comment:
```suggestion
The Stylitics widgets require the HTML page to have UTF-8 encoding enabled. Simply add the following to the `<head>` tag of your HTML.
```

Self-closing the tag is confusing since you won't be able to put a <meta> tag in it like that.